esphome/components/ota/ota_backend_esp_idf.h

```diff
@@ -20,6 +21,8 @@ class IDFOTABackend : public OTABackend {
  private:
   esp_ota_handle_t update_handle_{0};
   const esp_partition_t *partition_;
+  MD5Digest md5_{};
+  const char *expected_bin_md5_[32];
```

Comment:
Was this intended to be a pointer-to-array (which is pointer-to-pointer)? It doesn't seem to be used as such.
```suggestion
  const char expected_bin_md5_[32];
```

Reply:
Nope, that was just sleep typing ;-) I'll fix that.

Reply:
Fixed.